src/components/Projects/WBS/WBSDetail/components/TagsSearch.jsx

```diff
@@ -0,0 +1,84 @@
+import React, { useEffect, useState } from 'react';
+import TagSent from './TagSent';
+import './TagsSearch.css';
+
+function TagsSearch({ placeholder, members, addResources, removeResource, resourceItems }) {
+  const [isHidden, setIsHidden] = useState(false);
+  const [filteredData, setFilteredData] = useState([]);
+  const [selectedIndex, setSelectedIndex] = useState(0);
+
+  const handleClick = (event) => {
+    members.map(member =>{ 
+      if(`${member.firstName} ${member.lastName}` == event.target.innerText) {
+        addResources(member._id, member.firstName, member.lastName);
+        console.log(resourceItems)
```

Comment:
not sure if you meant to keep the console.log

Reply:
Nice catch, Crystal! I'm going to delete it. Thank you!